cset-workflow/meta/rose-meta.conf

```diff
@@ -694,3 +728,11 @@ description=Extracts data required for, and calculates the inflow properties dia
 help=See includes/deterministic_plot_inflow_properties.cylc
 type=python_boolean
 compulsory=true
+
+[template variables=DOMAIN_HISTOGRAM_SERIES_FIELD]
+ns=Diagnostics
+description=Create a series of histogram plots for selected pressure levels for each cycle time.
+help=See includes/deterministic_domain_histogram_series.cylc
+type=python_boolean
```

Comment:
should this have some trigger fields so that it knows what is being plotted?

Reply:
For now I have not included trigger fields. Instead histograms will be plotted for all chosen levels and variables. I suggest that we review the different diagnostics in a separate ticket and decide how we deal with different fields being plotted by different diagnostics.

Reply:
I have added a trigger field to choose the type of histogram_plot and preset it to a line style